frontend/src/components/Public.vue

```diff
@@ -0,0 +1,345 @@
+      this.$store.state.Vehicles.forEach((veh: Vehicle) => {
+        if (this.Map !== undefined) {
+          veh.addToMap(this.Map);
+        }
+      });
+    },
+    showUserLocation() {
+      const userIcon = L.icon({
+        iconUrl: 'static/images/user.svg',
+
+        iconSize:     [12, 12], // size of the icon
+        iconAnchor:   [6, 6], // point of the icon which will correspond to marker's location
+        shadowAnchor: [6, 6],  // the same for the shadow
+        popupAnchor:  [0, 0], // point from which the popup should open relative to the iconAnchor
+      });
+
+      if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(((position) => {
+          const locationMarker = {
+                name: 'You are here',
+                marker: L.marker([position.coords.latitude, position.coords.longitude], {
+                    icon: userIcon,
+                    zIndexOffset: 1000,
+                }),
+          };
+          locationMarker.marker.bindPopup(locationMarker.name);
+          if (this.Map !== undefined) {
+            locationMarker.marker.addTo(this.Map);
+          }
+        }) as PositionCallback);
+      }
+    },
+  },
+  components: {
+    dropdown,
+    messagebox,
+  },
+});
+</script>
+
+<style lang="scss">
+.caution-circle {
+  float:right;
+  width:10px;
+  height:10px;
+  background-color:orange;
+  border-radius:50%;
+}
+.pulsate {
+  float:right;
+  width:10px;
+  height:10px;
+  background-color:blue;
+  border-radius:50%;
+  animation: pulsate 2.5s ease-out;
+  animation-iteration-count: infinite;
+}
+@keyframes pulsate {
+  0% {
+    opacity: 0;
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+
+  }
+}
+.livebox{
+    p {
+      margin-right: 5px;
+    }
+    position: absolute;
+    height: 20px;
+    right: 10px;
+    top: 40px;
+    background-color: rgba(255, 255, 255, 0.88); 
+    border-color: black;
+    border-style: solid;
+    border-width: 1px;
+    padding-left: 4px;   
+    padding-right: 4px;   
+    justify-self: flex-end;
+    display: flex;
+    flex-flow: row wrap;
+    align-content: space-between;
+    align-items: center;
+    justify-content: space-around;
+}
+
+#mymap{
+    height: 100%;
+    width: 100%;
+    position: relative;
+    filter: invert(0);
+    
+}
+
+.titleBar {
+    height: 34px;
+    float: none;
+    position: absolute;
+    z-index: 1;
+    display: flex;
+    align-content: space-around;
+    justify-content: space-between;
+    flex-flow: row;
+    width: 100%;
+    background-color: rgba(255, 255, 255, 0.88);
+    box-shadow: 0 -5px 10px rgba(0, 0, 0, 0.8);
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+
+    & .titleContent {
+      height: 100%;
+      z-index: 1;
+      width: auto;
+      list-style: none;
+      position: relative;
+      top: 0px;
+      margin: 0px;
+      padding: 0px;
+
+      & .title {
```

Comment:
The title should be a little bigger so that it stands out a little more from the menu items since the sizes are all kinda similar (but I like how it's centered vertically here as it is not in the current version)

Reply:
updated